packages/lab/src/description-dotted-line-item/DescriptionDottedLineItem.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { Meta } from '@storybook/react';
+import { TextBox } from '../text-box/TextBox';
+import { DescriptionDottedLineItem } from './DescriptionDottedLineItem';
+
+export default {
+  title: 'Lab/DescriptionDottedLineItem',
```

Comment:
@timurhamitov What do you think of this name?

Reply:
@doniyor2109 I think, It sounds too technical. Most of the items are called to represent their usage.
Maybe we can find something for this item. 
It looks like a Table of Content in the books. But `DescriptionTableContentItem` sounds wrong=)
Maybe just `DescriptionLineItem`.

Reply:
Agree